docs/streams/developer-guide/dsl-api.html

```diff
@@ -1656,7 +1663,10 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                             <strong>It is the responsibility of the user to ensure data co-partitioning when joining</strong>.</p>
                         <div class="admonition tip">
                             <p><b>Tip</b></p>
-                            <p class="last">If possible, consider using <a class="reference internal" href="#streams_concepts_globalktable"><span class="std std-ref">global tables</span></a> (<code class="docutils literal"><span class="pre">GlobalKTable</span></code>) for joining because they do not require data co-partitioning.</p>
+                            <p class="last">
+                            A KTable-to-KTable Foreign-Key Join does not require co-partitioning.
+
+                            Also, if possible, consider using <a class="reference internal" href="#streams_concepts_globalktable"><span class="std std-ref">global tables</span></a> (<code class="docutils literal"><span class="pre">GlobalKTable</span></code>) for joining because they do not require data co-partitioning.</p>
```

Comment:
Not sure if we should add this... using global-table will provide quite different semantics.

Reply:
True. I was just trying to get the no-co-partitioning requirement included in the same area.